environment

```diff
@@ -18,14 +18,10 @@ DSS_SECRETS_STORE="${DSS_PLATFORM}/dss"
 
 DSS_INFRA_TAG_PROJECT="${DSS_PLATFORM}-dss"
 DSS_INFRA_TAG_SERVICE="dss"
-DSS_INFRA_TAG_OWNER="team-redwood-group@ucsc.edui"
-
-#not sure if these are needed
-OIDC_AUDIENCE=https://dss.dev.ucsc-cgp-redwood.org
-AUTH_URL=https://auth.ucsc.ucsc-cgp-redwood.org
-OPENID_PROVIDER=https://dev-4lyab62k.auth0.com/
-OIDC_EMAIL_CLAIM="${OIDC_AUDIENCE}/email"
-OIDC_GROUP_CLAIM="${OIDC_AUDIENCE}/group"
+DSS_INFRA_TAG_OWNER="team-redwood-group@ucsc.edu"
+
+# dont use interpolation on the stage here, its going to change the claims that are set in the auth0_rule
```

Comment:
👍

Reply:
its for us in the future tbh.
in the future we gonna look back and say, `thx amar`

Reply:
May want to mention OIDC_AUDIENCE variable should have a trailing slash